Documentation/microsoft-team.md

```diff
@@ -0,0 +1,23 @@
+# Microsoft Team Onboarding
+
+If you are new to Microsoft, you will need to onboard into various GitHub projects in order to get your work done.
+
+## Getting Permissions
+
+There are two main teams to join to get access to Microsoft .NET-related projects.
```

Comment:
We should mention linking the [corpnet account to GitHub](https://repos.opensource.microsoft.com/link)